release.json

```diff
@@ -10,6 +10,17 @@
 		"OMNIBUS_RUBY_BRANCH": "agent-v5",
 		"OMNIBUS_SOFTWARE_BRANCH": "agent-v5"
 	},
+	"5.32.2-rc.2": {
+		"AGENT_VERSION": "5.32.2",
+		"AGENT_BRANCH": "5.32.2-rc.1",
+		"AGENT6_BRANCH": "6.11.0-rc.1",
+		"INTEGRATIONS_CORE_BRANCH": "agent-v5",
+		"TRACE_AGENT_BRANCH": "6.11.0-rc.1",
+		"TRACE_AGENT_ADD_BUILD_VARS": "true",
+		"PROCESS_AGENT_BRANCH": "6.10.0",
+		"OMNIBUS_RUBY_BRANCH": "agent-v5",
+		"OMNIBUS_SOFTWARE_BRANCH": "agent-v5"
+	},
```

Comment:
Correctly using `agent-v5` in `INTEGRATIONS_CORE_BRANCH` instead of `6.11.0-rc.1`